Streetcode/Streetcode.BLL/Mapping/AdditionalContent/Coordinates/CoordinateProfile.cs

```diff
@@ -0,0 +1,13 @@
+using AutoMapper;
```

Comment:
Fix indentation problems in all files from this commit (Unnecessary blank lines, unconventional inheritance spacing, etc).